src/components/beers.js

```diff
@@ -0,0 +1,48 @@
+import React,{useState, useEffect} from 'react';
```

Comment:
Same here, you need to rename this component `Beers.js` for the import in `App.js` to work.